Calculator app milestone two

che30 opened the pull request:
In this project I implemented the following: 
- Created a components directory in the src folder.
- Create four files in there, each one representing one React component
- Implemented the App component
- Implemented the Display component
- Implemented the Button component
- Implemented the ButtonPanel component

che30 committed:
Add the following
Add an app file to hold components
Adda button file to render buttons
Add a buttonpanel file to call single buttons
Add a result file to display results

che30 committed:
Add styles to the index.css file

che30 committed:
Fix styleint errors

che30 committed:
Delete package-lock.json

Torres-ssf commented on src/component/Buttonpanel.js:
```diff
@@ -0,0 +1,38 @@
+  <div className="buttons-div">
+    <div className="board-row">
+      <Button value="AC" />
+      <Button value="+" />
```
- [ ] The second digit for the first row should be `+/-`
![Screenshot from 2021-05-10 19-21-20](https://user-images.githubusercontent.com/30977845/117731683-ee05dc80-b1c4-11eb-866e-439679ad38ab.png)

Torres-ssf commented on src/component/Result.js:
```diff
@@ -0,0 +1,21 @@
+/* eslint-disable react/prop-types */
+/* eslint-disable react/destructuring-assignment */
```
- [ ] Let's remove these eslint disable lines

Torres-ssf commented on src/component/Result.js:
```diff
@@ -0,0 +1,21 @@
+      {props.result}
+    </div>
+  );
+}
```
- [ ] As it is required by the instructions, we should use the name Display for this component instead.

Torres-ssf commented on src/component/Result.js:
```diff
@@ -0,0 +1,21 @@
+};
+Result.propTypes = {
+  result: PropTypes.number,
+};
```
- [ ] result props should be a string
- [ ] The default value should be '0' instead of 0
![Screenshot from 2021-05-10 19-18-56](https://user-images.githubusercontent.com/30977845/117731489-98c9cb00-b1c4-11eb-8634-c7112f5d24fb.png)

Torres-ssf reviewed (changes requested):
### Changes required :muscle: 
Hello @che30 :hand: 
Great job on this milestone.
We need to make some changes before moving on.

- [ ] Check the comments below

Happy coding :computer: 
**Sérgio Torres**

che30 committed:
Fix calculator button arrangments

che30 committed:
Add the following
Change the name of result file to display
Make the defalut prop type to be string

uche-inyama reviewed (changes requested):
## Changes Required

### Hi che30 :slightly_smiling_face: 

There is one update you missed from the last review. Eslint should not be disabled, kindly fix the errors in **src/component/display.js** file.

That's it, make the change, then submit another form for code review.


### Have a great day :man_technologist:  

#### ``` Uchechukwu Henry Inyama ```

che30 committed:
Remove disabled eslint

bellom reviewed (approved):
## Hi @che30  :clap: 

**Great work making all the necessary corrections from the previous review.**

# Status: Approved 👍 

## Congratulation :+1:  
_Go ahead and merge to dev and don't forget to submit the project completion form.
Good luck in the next milestone :slightly_smiling_face:_ 

**Best,**

**Oluwaseun Bello**
**_Slack Handle - @sean bellom_**
